README.md

```diff
@@ -4,24 +4,32 @@
 
 # Sinopia Indexing Pipeline
 
-This is the repository for the Sinopia Indexing Pipeline. The pipeline is a Node application that listens for messages (sent to a queue via STOMP), and for each message (a W3C Activity Streams message):
+This is the repository for the Sinopia Indexing Pipeline. The pipeline is a Node application that
+responds to a MongoDB [Change Streams](https://docs.mongodb.com/manual/changeStreams/) and
+indexes parts of the document into Elasticsearch:
```

Comment:
Open to `ElasticSearch` for consistency w/ line 12?

Reply:
Done, fixed with latest commit.